packages/shared/src/schemas.ts

```diff
@@ -10,12 +10,34 @@ export const quoteQuerySchema = z.object({
 
 export type QuoteQueryParams = z.infer<typeof quoteQuerySchema>;
 
+export type ByteString = string | `0x${string}`;
+export interface CcmMetadata {
+  gas_budget: string | number;
```

Comment:
This might actually be:
```suggestion
  gas_budget: `0x${string}` | number;
```